Wild Talents 2nd Edition/wt2.html

```diff
@@ -0,0 +1,3042 @@
+<h1 class="sheet-title" align="center">Wild Talents</h1 >
+<input type="radio" name="attr_tab" class="sheet-tab sheet-tab1" value="1" checked="checked" /><span title="GEN"></span>
+<input type="radio" name="attr_tab" class="sheet-tab sheet-tab2" value="2" checked="checked" /><span title="SKIL"></span>
+<input type="radio" name="attr_tab" class="sheet-tab sheet-tab3" value="3" checked="checked" /><span title="PWR"></span>
+<input type="radio" name="attr_tab" class="sheet-tab sheet-tab4" value="4" checked="checked" /><span title="BIO"></span>
+<input type="radio" name="attr_tab" class="sheet-tab sheet-tab5" value="5" checked="checked" /><span title="INV"></span>
+<div class="sheet-tab-content sheet-tab1">
+    
+                <!--This table contains base character information-->
+<table class="sheet-nametable">
```

Comment:
Greetings @Kumatei 

We no longer accept character sheets that use `<table>`s for layouts. These tables will need to be removed in order to meet the [Building Character Sheet Standards](https://wiki.roll20.net/Building_Character_Sheets)

Thank you,
Cassie

Reply:
Hi, can you explain further what I should do? As I'm sure you are well aware based on how janky I made this, I don't know what I'm doing and mostly just wing it.

Reply:
You'll need to remove all the `<table>` references and their children then use things like `<div>` and `<label>`. A good example sheet to look at for reference is the [kitchensink](https://github.com/Roll20/roll20-character-sheets/blob/master/kitchensink/kitchensink.html).